### ISSUE TITLE
Did other shots try? Like 20shot

### ISSUE THREAD

**Issue by luoyq6** [LiewFeng/imTED#5]
**Hello, is there any difference between the configs files of 10shot and 30shot? Or I want to test the effect in 20shot. What should I change**

**Comment by LiewFeng** [comment]
1. We don't conduct experiments on 20-shot setting. You can try it. We are glad to get your feedback on other shots.
2. The difference is the path of json file for 10-shot or 30-shot setting. All hyper-parameters are the same for 10-shot and 30-shot setting. They are supposed to work for 20 shot setting.
3. Thus, you only need to modify the path of json file to 20-shot setting. Also, modify the learning rate and which epoch to decay learning rate may help you get better performance.
